website/content/docs/release-notes/consul/v1_15_x.mdx

```diff
@@ -68,6 +68,18 @@ For more detailed information, please refer to the [upgrade details page](/consu
 
 The following issues are known to exist in the v1.15.0 release:
 
+- For v1.15.0, Consul is reporting newer releases of Envoy - 1.25.1, for example - as not supported, even though these versions are listed as valid in the [Envoy compatilibity matrix]( https://developer.hashicorp.com/consul/docs/connect/proxies/envoy#envoy-and-consul-client-agent). The following error would result for newer versions of Envoy:
+
+  ```log hideClipboard
+  Envoy version 1.25.1 is not supported. If there is a reason you need to use this version of envoy use the ignore-envoy-compatibility flag. Using an unsupported version of Envoy is not recommended and your experience may vary.
+  ```
+  
+  The workaround to resolve this issue until Consul 1.15.1 would be to run the client agents with the new `ingore-envoy-compatiblity` flag: 
+  
+  ```log hideClipboard
+  consul connect envoy --ignore-envoy-compatibility`
+  ```
```

Comment:
since users can c/p this command into their terminal

````suggestion
  ```shell-session
  $ consul connect envoy --ignore-envoy-compatibility
  ```
````